cmd/core/main.go

```diff
@@ -135,40 +134,28 @@ func main() {
 		setupLog.Error(err, "unable to create manager")
 		os.Exit(1)
 	}
-
 	ns := util.PodNamespace(systemNamespace)
-	storageURL, err := url.Parse(fmt.Sprintf("%s/bundles/", httpExternalAddr))
-	if err != nil {
-		setupLog.Error(err, "unable to parse bundle content server URL")
-		os.Exit(1)
-	}
-
-	localStorage := &storage.LocalDirectory{
-		RootDirectory: provisionerStorageDirectory,
-		URL:           *storageURL,
-	}
-
+	var bundleCA io.Reader
 	var rootCAs *x509.CertPool
 	if bundleCAFile != "" {
+		bundleCA, err = os.Open(bundleCAFile)
+		if err != nil {
+			setupLog.Error(err, "unable to create storage")
+			os.Exit(1)
+		}
 		var err error
-		if rootCAs, err = util.LoadCertPool(bundleCAFile); err != nil {
+		if rootCAs, err = util.LoadCertPool(bundleCA); err != nil {
 			setupLog.Error(err, "unable to parse bundle certificate authority file")
 			os.Exit(1)
 		}
 	}
-
-	httpLoader := storage.NewHTTP(
-		storage.WithRootCAs(rootCAs),
-		storage.WithBearerToken(cfg.BearerToken),
-	)
-	bundleStorage := storage.WithFallbackLoader(localStorage, httpLoader)
-
-	// NOTE: AddMetricsExtraHandler isn't actually metrics-specific. We can run
-	// whatever handlers we want on the existing webserver that
-	// controller-runtime runs when MetricsBindAddress is configured on the
-	// manager.
```

Comment:
It looks like this comment was cut during these changes. Do we feel like this comment doesn't provide any value anymore, or isn't still relevant?

Reply:
Oh, I restore it.  Thanks!